experiments/modern/src/App.js

```diff
@@ -31,6 +31,16 @@ const IGNORE_IDS = new Set([
 
 const SkinContext = React.createContext(null);
 
+async function loadImage (imgUrl) {
+  return await new Promise(resolve => {
+    const img = new Image();
+    img.addEventListener("load", function() {
+      resolve([img.width, img.height]);
```

Comment:
How about we make this just return the image itself? If we want one which just returns the size, it could call this under the hood

Reply:
sounds good!